crates/pop-cli/tests/pop_add_to_runtime_pallet.rs

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use assert_cmd::Command;
+use similar::{ChangeTag, TextDiff};
+
+#[test]
+fn pop_add_to_runtime_pallet_runtime_macro_v2_works() {
```

Comment:
Like the previous case, it correctly shows that the required lines are being injected. However, it doesn't yet confirm that everything works end-to-end, for example, that it compiles. Since the goal of an e2e test is to validate the full flow, including `pop build` would be great.